hudi-cli/src/main/java/org/apache/hudi/cli/commands/CommitsCommand.java

```diff
@@ -442,8 +442,8 @@ public String compareCommits(@CliOption(key = {"path"}, help = "Path of the tabl
     }
   }
 
-  @CliCommand(value = "commits sync", help = "Compare commits with another Hoodie table")
-  public String syncCommits(@CliOption(key = {"path"}, help = "Path of the table to compare to") final String path) {
+  @CliCommand(value = "commits sync", help = "Sync commits with another Hoodie table")
+  public String syncCommits(@CliOption(key = {"path"}, help = "Path of the table to sync to") final String path) {
```

Comment:
Looks like this CLI command does not have the actual implementation.  Still good to fix the docs due to copy and paste.

Reply:
Yes, I just came into learnning hudi, I don’t know a lot